src/graph/backend/dnnl/passes/transform.cpp

```diff
@@ -2297,6 +2298,149 @@ status_t binary_canonicalization(std::shared_ptr<subgraph_t> &sg) {
     return infer_shape(sg);
 }
 
+status_t decompose_select_to_multiple_binary_ops(
```

Comment:
nit: I feel `ops` already implies `multiple`.

```suggestion
status_t decompose_select_to_binary_ops(
```

Reply:
Fixed.